packages/playback-core/package.json

```diff
@@ -63,6 +63,7 @@
     "mux-embed": "~5.2.0"
   },
   "devDependencies": {
+    "@mux/esbuilder": "0.1.0",
```

Comment:
NOTE: Using (an updated) esbuilder for playback-core as well to account for language json files.